src/view/about-info.inc

```diff
@@ -1,7 +1,8 @@
 {# This is the template for the about page. You can enter any html here and show different language versions by using the lang value in a twig conditional as demonstrated below. #}
 {% block content %}
-  {% if request.lang == 'fi' %}
-    <h4>Tietoja sanastoselaimesta</h4>
+<div id="about-content">
+	{% if request.lang == 'fi' %}
```

Comment:
SonarCloud complains about tab characters here, please replace with spaces to follow PSR2 conventions.

Reply:
![image](https://github.com/NatLibFi/Skosmos/assets/50104570/1fb93581-c725-4f3c-a507-3bb72226cdb7)